### ISSUE TITLE
Ignoring classes in the class_dict.csv

### ISSUE THREAD

**Issue by k22jung** [GeorgeSeif/Semantic-Segmentation-Suite#172]
- **What are your command line arguments?:**
```
 python3 train.py --num_epochs 10 --checkpoint_step 1 --validation_step 1 --dataset /path/to/dataset --crop_width 512 --crop_height 512 --batch_size 2 --num_val_images 100 --h_flip 1 --v_flip 0 --brightness 0.5 --rotation 10 --model Encoder-Decoder-Skip --frontend MobileNetV2
```
- **Have you written any custom code?**:
No.
- **What have you done to try and solve this issue?:**
- **TensorFlow version?**:
1.10.1

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug or a feature request.

I am trying to train on the [mapillary](https://www.mapillary.com/) dataset. There are a total of 67 classes each with their respective color. However, I am only interested in training on 3/67 classes out of the bunch. How should I go about pre-processing the labels and/or handling multiple unlabelled classes in the CSV? There are multiple colored labels in the data I will not need for training, so I wasn't sure how the program would process the `class_dict.csv`.

**Comment by SchotmanG** [comment]
Maybe there is a better solution but you could simply do this: Convert all pixels of classes in which your not interested in [0,0,0] and make this the void/background class. So you would end up with 4 classes in the dataset and in the class_dict.csv.

I didn't see any straightforward implementation of ignoring classes or something like this.

**Comment by k22jung** [comment]
Thank you for the input, what you mentioned worked for me. I wrote a Numpy script to take each label and replace the colors associated with unnecessary classes to be black and any other classes to be left alone. This is the class dict I used that had only the classes I wanted:

```

name | r | g | b
Void | 0 | 0 | 0
road-marking-crosswalk | 200 | 128 | 128
road-marking-general | 63 | 234 | 86
Road | 128 | 64 | 128


```

I will now close this issue.
